identify/extensions.py

```diff
@@ -6,6 +6,7 @@
     'avif': {'binary', 'image', 'avif'},
     'bash': {'text', 'shell', 'bash'},
     'bat': {'text', 'batch'},
+    'bats': {'text', 'shell', 'bash'},
```

Comment:
this should probably have `bats` as well so that if bats-specific linting is needed it can be done against that `type`

Reply:
I hope I'm getting this correctly. 

I didn't add `bats` as a `type` due to the fact that there is no `bats` specific linter etc but everything is based on bash. However, there can good use cases in the future. So, the next commit incudes that, too.